pages/learn/deploy/deployment.md

```diff
@@ -15,7 +15,7 @@ __Note:__ The resin.io git repository is not intended as a code hosting solution
 
 ## The build server
 
-The build server is an incredibly powerful tool which cross-compiles code for the target device on our (far more powerful) server. This gives you the ability to compile a gnarly dependency tree in seconds on the server rather than minutes or even hours to build on your device.
+The build server is an incredibly powerful tool which cross-compiles code for the target device type or architecture on our (far more powerful) server. This gives you the ability to compile a gnarly dependency tree in seconds on the server rather than minutes or even hours to build on your device.
```

Comment:
Should this be "on our (far more powerful) servers" ? (IIRC there's different build servers for x86 and ARM?)